00 SFC Boilerplate/README.md

```diff
@@ -0,0 +1,319 @@
+# 00 SFC Boilerplate
+
+In this sample we are going to setup a web project that can be easily managed
+by webpack.
+
+We won't install anything related to Vue.js, just some basic plumbing.
+
+We will setup an initial <abbr title="Node.js package manager, a package manager for the JavaScript runtime environment Node.js">npm</abbr> project and give support to TypeScript. Then we will create a **helloworld.ts** sample.
+
+Summary steps:
+
+- Prerequisites: Install Node.js
+- Initialize **package.json** (with `npm init`)
+- Install:
+  - Webpack and webpack-dev-server.
+  - TypeScript.
+  - Bootstrap.
+- Setup **webpack.config.js**
+- Create a test js file.
+- Create a simple HTML file.
+
+# Prerequisites
+
+Install [Node.js and npm](https://nodejs.org/en/) (v6.x.x or higher) if they are not already installed on your computer.
+
+> Verify that you are running at least node v6.x.x and npm 3.x.x by running `node -v` and `npm -v` in a terminal/console window. Older versions may produce errors.
+
+## Steps to build it
+
+- Create and navigate to the folder where you are going to create the empty project.
+
+- Execute `npm init`, you will be prompted to answer some information about the project (e.g. set name to _sample-vue-js_ and description to _Sample working with TypeScript and Webpack_).
+Once you have successfully fullfilled them a **package.json** file we will generated.
+
+ ```
+ npm init
+ ```
+
+- Install **webpack** as a development dependency.
+
+ ```
+ npm install webpack webpack-cli --save-dev
+ ```
+
+- Install **webpack-dev-server** locally, as a development dependency (the reason to install it locally instead of globally is for it to be easy to setup, e.g. It can be launched on a clean machine without having to install anything globally but nodejs).
+
+ ```
+ npm install webpack-dev-server --save-dev
+ ```
+
+- Let's install a list of plugins and loaders that will add powers to our webpack configuration (handling <abbr title="Cascading Style Sheets">CSS</abbr>, TypeScript...).
+
+ ```
+ npm install ts-loader fork-ts-checker-webpack-plugin css-loader file-loader html-webpack-plugin mini-css-extract-plugin url-loader  --save-dev
```

Comment:
Why do you preferer install ts-loader + fork-ts-checker-webpack-plugin instead of awesome-typescript-loader? Last one has `const { CheckerPlugin } = require('awesome-typescript-loader')` to do same as fork-ts